.github/workflows/tests.yml

```
@@ -22,7 +22,7 @@ jobs:
       - name: Set up Python and uv
         uses: drivendataorg/setup-python-uv-action@v1
         with:
-          python-version: 3.8
+          python-version: [3.11]
```

```suggestion
          python-version: 3.11
```
shouldn't be a sequence

https://github.com/drivendataorg/zamba/actions/runs/10734680787/workflow